lib/mpl_toolkits/axisartist/axis_artist.py

```diff
@@ -658,18 +715,18 @@ def __init__(self, axes,
 
     def set_axis_direction(self, axis_direction):
         """
-        Adjust the direction, text angle, text alignment of
-        ticklabels, labels following the matplotlib convention for
+        Adjust the direction, text angle, and text alignment of
+        ticklabels, Labels following the Matplotlib convention for
         the rectangle axes.
```

Comment:
I don't quite understand the second sentence (I assume it should be a separate sentence, but there's a comma there...)

Reply:
I think I got it correct now, but I clearly did something strange the first time.